pysilcam/silcamgui/guicalcs.py

```diff
@@ -248,7 +247,7 @@ def annotate(datadir, filename):
 def silcview(datadir):
     files = [os.path.join(datadir, f) for f in
             sorted(os.listdir(datadir))
-            if f.endswith('.silc') or f.endswith('.bmp')]
+            if f.endswith('.silc') or f.endswith('.bmp') or f.endswith('.silc_mono')]
```

Comment:
E128: continuation line under-indented for visual indent  <!-- Automatically posted by Lintly -->